C/README.md

```diff
@@ -66,6 +66,8 @@ Format:
 
 -[Merge Sort](Mergesort.c)
 
+-[Minimum number of denominations using Greedy algo approach](MinDenominations_GreedyAlgo.c)
```

Comment:
I would add Indian currency here to be specific.
Thanks.

Reply:
Sure, I'm updating the readme.md file